getting-started/deployment_guide.md

```diff
@@ -284,4 +289,124 @@ kubectl exec -it $POD -- bash -c "chroot"
 ```
 Any binary other than `bash` and `nginx` would be permission denied.
 
+If you don't see Permission denied please refer [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#debug-kubearmor-installation-issue-in-dockerized-kubernetes-environment) to debug this issue
+
+</details>
+
+<details>
+<summary><h4>Debug KubeArmor installation issue</h4></summary>
+In certain scenarios, the expected behavior of KubeArmor might not be observed. One way to investigate this is by using the KubeArmor Command Line Interface (CLI) utility, commonly referred to as [karmor cli](https://github.com/kubearmor/kubearmor-client). 
+
+To check the status and configuration of KubeArmor, you can use the following command:
+
+```
+karmor probe
+```
+
+<img src="../.gitbook/assets/kubearmor_install_issue.png" width="768" class="center" alt="Karmor logs">
+
+When executing this command, check the output for the value of **ActiveLSM** field, if it is not assigned any value, it means that no active LSM is available for KubeArmor to enforce policies. Under normal circumstances, this value should be assigned a specific Linux Security Module (LSM) that KubeArmor uses to enforce security policies. Additionally, ensure that the **Container Security** field is set to true.
+
+However, there are situations where ActiveLSM might not be assigned any value. This situation indicates that Kubearmor is unable to identify the appropriate LSM in a environment, which is commonly used in Kubernetes setups.
+
+To address this issue, KubeArmor provides a solution involving the use of BPF-LSM. BPF (Berkeley Packet Filter) is a technology that allows efficient packet filtering in the Linux kernel. Enabling support for BPF LSM ensures that KubeArmor can apply and enforce policies as expected in Dockerized environments associated with Kubernetes. Please note that BPFLSM is only available on kernel versions above 5.8 or on RHEL distros > 8.5.
+
+So we need to enable [bpf-lsm](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/FAQ.md#checking-and-enabling-support-for-bpf-lsm) for Kubearmor to apply and enforce policies as expected.
+
+You can also enable AppArmor if you want to use it as a security module to enforce KubeArmor policies, please refer [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/FAQ.md#using-kubearmor-with-kind-clusters). There is a chance that neither AppArmor nor BPF-LSM is enabled on some nodes. 
+
+**We can include the following updater script which automatically detects and installs BPFLSM/AppArmor whichever is needed in kubernetes worker nodes.**
+
+```
+apiVersion: v1
+items:
+- apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
```

Comment:
Please add it as a file.

Please clean up the resource, it's a Daemonset List, it should be a normal daemonset.